<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SingleSizeShowWidget</class>
 <widget class="QWidget" name="SingleSizeShowWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1804</width>
    <height>453</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>40</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>光场</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>产品轮廓</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>990</x>
     <y>60</y>
     <width>811</width>
     <height>341</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>序号</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>时间</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>类型</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>等级</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>长X (mm)</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>宽Y (mm)</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>边距X (mm)</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>边距Y (mm)</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>60</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graphicsView">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>601</width>
     <height>281</height>
    </rect>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graphicsView_2">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>80</y>
     <width>291</width>
     <height>281</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
